erpnext/projects/doctype/time_log/time_log.js

```diff
@@ -25,16 +25,34 @@ frappe.ui.form.on("Time Log", "refresh", function(frm) {
 
 
 // set to time if hours is updated
-frappe.ui.form.on("Time Log", "hours", function(frm) {
-	if(!frm.doc.from_time) {
-		frm.set_value("from_time", frappe.datetime.now_datetime());
```

Comment:
sorry, checked further, he  did check "Simplified Time Log" in update_time function that is his new function. So this code is necessary if "Simplified Time Log" is unchecked.